worlds/cccharles/__init__.py

```diff
@@ -0,0 +1,171 @@
+                classification = ItemClassification.progression
+            case "Journal" :
+                classification = ItemClassification.progression
+            case "Timed Dynamite" :
+                classification = ItemClassification.progression
+            case "Box of Rockets" :
+                classification = ItemClassification.progression
+            case "Breaker" :
+                classification = ItemClassification.progression
+            case "Broken Bob" :
+                classification = ItemClassification.progression
+            case "Employment Contracts" :
+                classification = ItemClassification.progression
+            case "Mob Camp Key" :
+                classification = ItemClassification.progression
+            case "Jar of Pickles" :
+                classification = ItemClassification.progression
+            case "Orange Paint Can" :
+                classification = ItemClassification.filler
+            case "Green Paint Can" :
+                classification = ItemClassification.filler
+            case "White Paint Can" :
+                classification = ItemClassification.filler
+            case "Pink Paint Can" :
+                classification = ItemClassification.filler
+            case "Grey Paint Can" :
+                classification = ItemClassification.filler
+            case "Blue Paint Can" :
+                classification = ItemClassification.filler
+            case "Black Paint Can" :
+                classification = ItemClassification.filler
+            case "Lime Paint Can" :
+                classification = ItemClassification.filler
+            case "Teal Paint Can" :
+                classification = ItemClassification.filler
+            case "Red Paint Can" :
+                classification = ItemClassification.filler
+            case "Purple Paint Can" :
+                classification = ItemClassification.filler
+            case "The Boomer" :
+                classification = ItemClassification.filler
+            case "Bob" :
+                classification = ItemClassification.filler
+            case "Green Egg" :
+                classification = ItemClassification.progression
+            case "Blue Egg" :
+                classification = ItemClassification.progression
+            case "Red Egg" :
+                classification = ItemClassification.progression
+            case "Remote Explosive" :
+                classification = ItemClassification.progression
+            case "Remote Explosive x8" :
+                classification = ItemClassification.progression
+            case "Temple Key" :
+                classification = ItemClassification.progression
+            case "Bug spray":
+                classification = ItemClassification.progression
+            case _: # Should not occur
+                classification = ItemClassification.filler
+
+        return CCCharlesItem(name, classification, item_id, self.player)
+    
```

Comment:
There's a whitespace error on this line. I can't even properly suggest a change in github for this, but it's there if you enjoy the peak nitpick change.